src/app/components/SearchResultItem.vue

```diff
@@ -8,7 +8,7 @@
         <img
           class="img mx-auto cursor-pointer w-full lazy"
           :src="placeholder"
-          :data-src="path"
+          :data-src="itemImagePath"
```

Comment:
same here, no need to know that this is an item, `imageSrs` or `imagePath` or `imageSource` might be enough